test/src/server/routes/auth.js

```diff
@@ -0,0 +1,65 @@
+import {createEditor, truncateEntities} from '../../../test-helpers/create-entities';
+import app from '../../../../src/server/app';
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import {get} from 'lodash';
+import orm from '../../../bookbrainz-data';
+
+
+chai.use(chaiHttp);
+const {expect} = chai;
+const {Editor} = orm;
+
+const deletedUserAttrib = {
+	areaId: null,
+	bio: '',
+	cachedMetabrainzName: null,
+	genderId: null,
+	metabrainzUserId: null,
+	revisionsApplied: 0,
+	revisionsReverted: 0,
+	titleUnlockId: null,
+	totalRevisions: 0
+};
+
+function verifyDeletedUser(user) {
+	if (!get(user, 'name', '').includes('Deleted User')) { return false; }
+	for (const key in deletedUserAttrib) {
+		if (Object.hasOwnProperty.call(deletedUserAttrib, key)) {
+			if (get(user, key) !== get(user, key, null)) {
+				return false;
+			}
+		}
+	}
+	return true;
+}
+describe('delete user', () => {
+	const adminEditorId = 1;
+	const adminEditorName = 'UserDeleter';
+	const userEditorId = 2;
+	const userEditorName = 'NormalUser1';
+	const otherEditorId = 3;
+	const otherEditorName = 'NormalUser2';
+	let agent;
+	before(async () => {
+		await createEditor(adminEditorId, {name: adminEditorName});
```

Comment:
This user shouldn't be necessary.
The UserDeleter special user is in the MusicBrainz database, not in the BookBrainz one, and we don't really interact with MB other than authenticate it